### ISSUE TITLE
Is it possible to save the image in any format other than WEBP?

### ISSUE THREAD

**Issue by amoebatron** [VectorSpaceLab/OmniGen#82]
Such as JPG or PNG?

**Comment by staoxiao** [comment]
Yes, the output is PIL image format, you can save it as png or jpg: `images[0].save("example_t2i.png")`

**Comment by amoebatron** [comment]
> Yes, the output is PIL image format, you can save it as png or jpg: `images[0].save("example_t2i.png")`

Okay in that case I must be doing something wrong because I only have the option of saving in WEBP. Running in PInokio.
